onnx/compose.py

```diff
@@ -484,12 +484,12 @@ def _prefixed(prefix: str, name: str) -> str:
             for e in n.output:
                 name_map[e] = _prefixed(prefix, e)
 
-    if rename_inputs:
+    if not rename_inputs:
         for entry in g.input:
-            name_map[entry.name] = _prefixed(prefix, entry.name)
-    if rename_outputs:
+            name_map[entry.name] = entry.name
+    if not rename_outputs:
```

Comment:
I would be more confident with this change if there is test composing twice the same graph and to see what happens with the conflicts.